### DISCUSSION TITLE
Your password must be changed.

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15237] [How to]
How to switch off dialog: Your password must be changed. To continue using the system please enter a new password.

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=Password&parent=Settings&view=Index
![image](https://user-images.githubusercontent.com/10198654/135397302-7368766e-eafd-4bec-8a24-3227201c414c.png)
